web/modules/custom/joinup_comment/joinup_comment.module

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * @file
+ * Joinup comment logic.
+ */
+
+use Drupal\comment\CommentInterface;
+use Drupal\Core\Entity\Display\EntityViewDisplayInterface;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\message\Entity\Message;
+
+/**
+ * Implements hook_ENTITY_TYPE_view_alter().
+ */
+function joinup_comment_view_alter(array &$build, EntityInterface $entity, EntityViewDisplayInterface $display) {
+  // Add contextual links to comments.
+  $build['#contextual_links']['comment'] = [
+    'route_parameters' => ['comment' => $entity->id()],
+    'metadata' => ['changed' => $entity->getChangedTime()],
+  ];
+}
+
+/**
+ * Implements hook_entity_delete().
+ */
+function joinup_comment_entity_delete(EntityInterface $entity) {
+  if (!$entity instanceof CommentInterface) {
+    return;
+  }
+
+  $view_builder = \Drupal::entityManager()->getViewBuilder($entity->getEntityTypeId());
+  $build = $view_builder->view($entity, 'full');
+  $html = \Drupal::service('renderer')->renderRoot($build);
```

Comment:
The idea is that this way I don't have to care about catching all data...
Anyway, I don't like this logger stuff.